### ISSUE TITLE
Struggling to start dark mode in new tabs

### ISSUE THREAD

**Issue by SimplePotato** [dailynowco/daily#31]
#### Bug Report

Explanation: When I open a new tab the daily starts and in the first two seconds shows the dark mode, but after that dark theme disappear (turns white) even when the dark theme is on. Also, the side menu is transparent. If I want to set back to the dark mode I have to turn on the light mode (where everything works fine) and then turn on the dark mode again (Now is fully functional as it should be).

This problem only appears when opening new tabs. 

Reproduce: *Steps to reproduce the bug if you know them*

Version:  1.9.3

Browser version: Firefox developer edition 66.0b10 (64-bit)

OS: Windows 10 Pro 1809

*Please add screenshots if possible or any other information that can help to fix the bug*

![screenshot 45](https://user-images.githubusercontent.com/34000272/53301807-8ff4b500-3857-11e9-91d9-323500d522af.png)

#### Feedback

I really love to use Daily. Thanks  👍

**Comment by idoshamun** [comment]
Hi @SimplePotato,

Thanks for reporting this bug, this situation is caused when an unexpected error occurs, it is not actually the bright theme.
In order to solve it I have two questions:
- Does it happen every time you open a new tab or once in a while?
- Could you share your console output please?

Glad to hear that you love our product, stay tuned we will have some news soon!

**Comment by SimplePotato** [comment]
Hi @idoshamun sorry to reply so late

1. Yes, this happens on every new tab.
2. Screenshot of the console...
![screenshot 47](https://user-images.githubusercontent.com/34000272/53521006-5205c400-3ad7-11e9-9eb8-a56de381df04.png)

**Comment by idoshamun** [comment]
Looks like you are using some kind of tracker/analytics blocker. Could you tell me which?
I will download it and fix the issue

**Comment by SimplePotato** [comment]
I fix the issue. 

First I disable every add-on with only Daily running. The problem was still there. Then I reinstall the Daily and tried it out with other add-ons still disabled. It started working again. Then I enable all the previously disabled add-ons, turned laptop OFF/ON and Daily is now working perfectly as it should.

If you're still interested in blockers I have installed. But I don't think is their fault...

- AdBlock by AdBlock
- Disconnect by Disconnect
- DuckDuckGo Privacy Essentials by DuckDuckGo
- I don't care about cookies by Kiko
- NoScript by Giorgio Maone
- TunnelBear by TunnelBear LLC

**A simple reinstallation of Daily add-on, fix the problem.**

I'm closing this issue since the problem has been fixed.
